src/components/Reports/GDPRPermissions/SubscriberGdpr.styles.js

```diff
@@ -1,8 +1,15 @@
 import styled from 'styled-components';
 import { fonts } from '../../styles/fonts';
 
-export const TextColumn = styled.div`
+const TextColumn = styled.div`
   p {
     font-size: ${fonts.fontLvl3};
   }
 `;
+
+const EmptyBox = styled.div`
+  padding: 48px;
+  width: 100%;
+`;
+
+export { TextColumn, EmptyBox };
```

Comment:
Just curiosity, is it better of using multiple exports like

```
export const TextColumn = styled.div`
// . . .
export const EmptyBox = styled.div`
// . . .
```

Reply:
@bravoy93 explained to me that it is to have more control about what we are exporting.